model-mesh-mlserver-adapter/server/server.go

```diff
@@ -226,9 +225,9 @@ func adaptNativeModelLayout(files []os.FileInfo, modelID, modelPath, schemaPath,
 		}
 		// special handling of the config file
 		if filename == mlserverRepositoryConfigFilename {
-			configJSON, err1 := ioutil.ReadFile(source)
+			configJSON, err1 := os.ReadFile(source)
 			if err1 != nil {
-				return fmt.Errorf("Could not read model config file %s: %w", source, err1)
+				return fmt.Errorf("could not read model config file %s: %w", source, err1)
```

Comment:
why changing to lower case?

Reply:
It is a Go lang style pattern: https://google.github.io/styleguide/go/decisions.html#error-strings

Reply:
ah, right, embedded errors versus log.error, thanks!